internal/service/fsx/backup_test.go

```diff
@@ -17,6 +18,7 @@ import (
 func TestAccFSxBackup_basic(t *testing.T) {
 	var backup fsx.Backup
 	resourceName := "aws_fsx_backup.test"
+	rName := fmt.Sprintf("tf_acc_test_%d", sdkacctest.RandInt())
```

Comment:
small change lets use builtin func for test names `sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)`

Reply:
Changing all rName to sdkacctest.RandomWithPrefix(acctest.ResourcePrefix) and creating new variable for vName to strings.Replace(sdkacctest.RandomWithPrefix(acctest.ResourcePrefix), "-", "_", -1) since ONTAP Volume name can only include underscore not dash.